interface/src/app/experiment/experimentDetail.view.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Simple view embedding an experiment-detail.
+ */
+angular.module('adage.experimentDetail.view', [
+  'ui.router',
+  'adage.experimentDetail',
+  'adage.sampleBin.addItem',
+  'adage.sampleBin.addItemAnalyze',
+  'adage.utils'
+])
+
+.config(['$stateProvider', function($stateProvider) {
+  $stateProvider.state('experiment', {
+    url: '/experiment?id',
+    views: {
+      main: {
+        templateUrl: 'experiment/experimentDetail.view.tpl.html',
+        controller: 'ExperimentDetailViewCtrl as ctrl'
+      }
+    },
+    data: {pageTitle: 'Experiment Detail'}
+  });
+}])
+
+.controller('ExperimentDetailViewCtrl', ['$stateParams', 'ApiBasePath',
+  function ExperimentDetailViewCtrl($stateParams, ApiBasePath) {
+    this.id = $stateParams.id;
```

Comment:
Does it make sense to use the same `var ctrl = this;` syntax that we use below? I think it is nice to a) keep it consistent with the other code, and b) keep good track of what we are tacking on to the `ctrl` object (since, as we have discussed before, `this` can point to different things within a segment of code in JS).

Reply:
SGTM. This might even be a bug because the stuff inside the `loaded` function is a callback so `this` is not well defined. Nice catch!

- [ ] need to use a closure variable to ensure we use the right `this` in `experimentDetail`